méli-mélo/th-choose-canada/choose-canada.html

```diff
@@ -0,0 +1,78 @@
+---
+title: Choose Canada theme
+dateModified: 2025-03-26
+description: Choose Canada CSS styles
+lang: en
+css:
+- style.css
+layout: no-container
+---
+
+<div class="bg-cover bg-center choose-canada-banner" data-bgimg-srcset="https://test.canada.ca/experimental/choose-canada/images/1920x315-2.jpg 1920w, https://wet-boew.github.io/vocab/wb/utilities#no-image 767w">
+	<div class="container">
+		<div class="well header-rwd">
+			<h1 property="name" id="wb-cont">Choose Canada.ca</h1>
+			<p>Be your most flag‑flying, maple leaf buying, local‑adventuring self.</p>
+			<p>It’s time to choose Canada!</p>
+		</div>
+	</div>
+</div>
+
+<div class="container mt-4">
+	<div class="row wb-eqht-grd">
+		<div class="col-sm-6 col-md-4">
+			<section class="choose-canada-card">
+				<img src="https://dummyimage.com/360x203/000000/FFFFFF.png" alt="" class="img-responsive full-width">
+				<div class="card-body">
+					<h2 class="card-title"><a href="#" class="stretched-link">Item 1 title</a></h2>
+					<p>Item 1 text lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+				</div>
+			</section>
```

Comment:
Not sure that the use of h2 here is appropriate based on the use case.

I suggest to use an h3 here and then group all the card into its own section identified with an h2.

I also suggest to add a following section after the list of card with some dummy text. like

```
<section class="well">
<h2>Lorem ipsum</h2>
<p>placeholder, lorem ipsum</p>
<ul>
  <li><a href="#">Link # 1</a></li>
  <li><a href="#">Link # 2</a></li>
  <li><a href="#">Link # 3</a></li>
</ul>
</section>
```

Reply:
We should complement those card with some other basic style. When i remove all the style it does create confusion about what define the group of the inner card item.

The large top margin on the heading does create undesirable effect.

See this sample:
![image](https://github.com/user-attachments/assets/9f0469f8-4e9f-4156-bba5-650572b38faa)

![image](https://github.com/user-attachments/assets/98ced52c-707a-4254-b5c8-e78cd860310a)

![image](https://github.com/user-attachments/assets/c98497e6-08df-44bd-8a90-bc64621c63ad)